0019-refactor-image-operations/README.md

```diff
@@ -0,0 +1,315 @@
+
+```
+<<[UNRESOLVED optional short context or usernames ]>>
+Stuff that is being argued.
+<<[/UNRESOLVED]>>
+```
+
+When editing ZEPs, aim for focused, single-topic PRs to keep discussions clear. If
+you disagree with a section, open a new PR with suggested changes.
+
+Each ZEP covers one "feature" or "enhancement" throughout its lifecycle. You don’t
+need a new ZEP for moving from beta to GA. If new details emerge, edit the existing
+ZEP. Once a feature is "implemented", major changes should go in new ZEPs.
+
+The latest instructions for this template can be found in [this repo](/NNNN-zep-template/README.md).
+
+**Note:** PRs to move a ZEP to `implementable`, or significant changes to an
+`implementable` ZEP, must be approved by all ZEP approvers. If an approver is no
+longer appropriate, updates to the list must be approved by the remaining approvers.
+-->
+
+# ZEP-0019: Refactor Image operations
+
+<!--
+Keep the title short simple and descriptive. It should clearly convey what
+the ZEP is going to cover.
+-->
+
+<!--
+A table of contents helps reviewers quickly navigate the ZEP and highlights
+any additional information provided beyond the standard ZEP template.
+-->
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories (Optional)](#user-stories-optional)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Test Plan](#test-plan)
+      - [Prerequisite testing updates](#prerequisite-testing-updates)
+      - [Unit tests](#unit-tests)
+      - [e2e tests](#e2e-tests)
+  - [Graduation Criteria](#graduation-criteria)
+  - [Upgrade / Downgrade Strategy](#upgrade--downgrade-strategy)
+  - [Version Skew Strategy](#version-skew-strategy)
+- [Implementation History](#implementation-history)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Summary
+
+<!--
+This section is key for creating high-quality, user-focused documentation
+like release notes or a roadmap. You should gather this info before
+implementation starts to keep the focus on development, not writing. ZEP
+editors should ensure the `Summary` is clear and useful for a broad audience.
+
+A good summary should be at least a paragraph long.
+
+Follow the [documentation style guide] for this section and the rest of the ZEP.
+Keep line lengths reasonable to make it easier for reviewers to provide
+feedback and reduce unnecessary changes.
+
+[documentation style guide]: https://docs.zarf.dev/contribute/style-guide/
+-->
+
+Zarf uses crane to pull and push container images. Crane has several bugs which cause issues within Zarf. Through switching to [oras-go](https://github.com/oras-project/oras-go) Zarf will resolve many of the issues it faces with Crane. Additionally, Zarf will reap benefits from using the same library for container image operations and Zarf OCI package operations.
+
+## Motivation
```

Comment:
I like all of the considerations for bugs and fixes for concrete asks -- I'm curious about what the architectural implications will be though. How does migrating over to oras affect our _codebase_, our datatypes and our operations?

One recommendation: reduce _concrete_ detail in the motivation section, focus it on the big ideas. Then shift detail over to the Proposal section.

Reply:
I agree there are too many concrete details in the motivation section atm. Mentioning the problems more broadly would be better, then going over the solutions in the design details. 

However, I'd like to avoid talking about the architecture / the codebase in the motivation section. I want the motivation to be more user focused. If this were a technical refactor to improve testability, code clarity, or architecture I think those discussions in the motivation would be more relevant.

Reply:
After reading the section summaries in the comments I do still feel like we want the issues in the motivation section, but I broke it out into sub sections and added more of a high level around "why" we are moving.